2021-07-23 16:52:38,175 INFO    MainThread:1696 [wandb_setup.py:_flush():69] setting env: {}
2021-07-23 16:52:38,175 INFO    MainThread:1696 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-23 16:52:38,175 INFO    MainThread:1696 [wandb_init.py:_log_setup():336] Logging user logs to /home/usuariohi/Documentos/Repos/MixedMaskRCNN/wandb/run-20210723_165238-26slhxtz/logs/debug.log
2021-07-23 16:52:38,176 INFO    MainThread:1696 [wandb_init.py:_log_setup():337] Logging internal logs to /home/usuariohi/Documentos/Repos/MixedMaskRCNN/wandb/run-20210723_165238-26slhxtz/logs/debug-internal.log
2021-07-23 16:52:38,176 INFO    MainThread:1696 [wandb_init.py:init():369] calling init triggers
2021-07-23 16:52:38,176 INFO    MainThread:1696 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {'test_set': '/run/user/1000/gvfs/smb-share:server=lxestudios.hospitalitaliano.net,share=pacs/T-Rx/TRx-v2/Experiments/2021-07-15_binary/testCSV.csv', 'experiment': '2021-07-15_binary', 'experiment_type': 'boxes', 'tested_epoch': 3, 'opacityies_as_binary': True, 'masks_as_boxes': True, 'costs_ratio': 1.0, 'expected_prevalence': 0.1, 'calculate_coco': False, 'calculate_classification': False, 'binary_classifier': None, 'adjust_new_binary_classifier': False, 'save_figures': 'heatmap', 'only_best_datasets': False, 'save_csv': False, 'view_in_window': False, 'loop': False, 'force_cpu': False}
2021-07-23 16:52:38,176 INFO    MainThread:1696 [wandb_init.py:init():418] starting backend
2021-07-23 16:52:38,176 INFO    MainThread:1696 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-07-23 16:52:38,224 INFO    MainThread:1696 [backend.py:ensure_launched():132] starting backend process...
2021-07-23 16:52:38,264 INFO    MainThread:1696 [backend.py:ensure_launched():137] started backend process with pid: 1794
2021-07-23 16:52:38,266 INFO    MainThread:1696 [wandb_init.py:init():423] backend started and connected
2021-07-23 16:52:38,271 INFO    MainThread:1696 [wandb_init.py:init():465] updated telemetry
2021-07-23 16:52:38,272 INFO    MainThread:1696 [wandb_init.py:init():484] communicating current version
2021-07-23 16:52:39,271 INFO    MainThread:1696 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.11.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-07-23 16:52:39,271 INFO    MainThread:1696 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-07-23 16:52:39,536 INFO    MainThread:1696 [wandb_init.py:init():522] starting run threads in backend
2021-07-23 16:52:40,115 WARNING MainThread:1696 [wandb_init.py:init():764] interrupted
Traceback (most recent call last):
  File "/home/usuariohi/anaconda3/envs/pytorch-opacidad/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 744, in init
    run = wi.init()
  File "/home/usuariohi/anaconda3/envs/pytorch-opacidad/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 525, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/usuariohi/anaconda3/envs/pytorch-opacidad/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 799, in communicate_run_start
    result = self._communicate(rec)
  File "/home/usuariohi/anaconda3/envs/pytorch-opacidad/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 537, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/usuariohi/anaconda3/envs/pytorch-opacidad/lib/python3.6/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/usuariohi/anaconda3/envs/pytorch-opacidad/lib/python3.6/threading.py", line 551, in wait
    signaled = self._cond.wait(timeout)
  File "/home/usuariohi/anaconda3/envs/pytorch-opacidad/lib/python3.6/threading.py", line 299, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
